resources/howtos/search_bar_for_docs.md

```diff
@@ -0,0 +1,201 @@
+
+### Configuration File
+
+The scraper needs a configuration file to know the content to be scraped thanks to the selectors (e.g. the HTML tag).
+
+Here is a basic configuration file:
+
+```json
+{
+  "index_uid": "docs",
+  "start_urls": ["https://www.example.com/doc/"],
+  "sitemap_urls": ["https://www.example.com/sitemap.xml"],
+  "stop_urls": [],
+  "selectors": {
+    "lvl0": {
+      "selector": ".docs-lvl0",
+      "global": true,
+      "default_value": "Documentation"
+    },
+    "lvl1": {
+      "selector": ".docs-lvl1",
+      "global": true,
+      "default_value": "Chapter"
+    },
+    "lvl2": ".docs-content .docs-lvl2",
+    "lvl3": ".docs-content .docs-lvl3",
+    "lvl4": ".docs-content .docs-lvl4",
+    "lvl5": ".docs-content .docs-lvl5",
+    "lvl6": ".docs-content .docs-lvl6",
+    "text": ".docs-content p, .docs-content li"
+  }
+}
+```
+
+The `index_uid` field is the index identifier in your MeiliSearch instance in which your website content is stored. The scraping tool will create a new index if it does not exist.
+
+The `docs-content` class is the main container of your textual content. Most of the time, this tag is a `<main>` or an `<article>` HTML element.
+
+`lvlX` selectors should use the standard title tags like `h1`, `h2`, `h3`, etc. You can also use static classes. Set a unique id or name attribute to these elements.
+
+Every searchable `lvl` elements outside this main documentation container (for instance, in a sidebar) must be `global` selectors. They will be globally picked up and injected to every record built from your page.
+
+If you use Vuepress for your documentation, you can check out the [configuration file](https://github.com/meilisearch/documentation/blob/master/.vuepress/scraper/config.json) we use in production.
+In our case, the main container is `theme-default-content` and the selector the titles and sub-titles are `h1`, `h2`...
+
+### Run the Scraper
+
+You can run the scraper with Docker:
+
+```bash
+$ docker run -t --rm \
+    -e MEILISEARCH_HOST_URL=<your-meilisearch-host-url> \
+    -e MEILISEARCH_API_KEY=<your-meilisearch-api-key> \
+    -v <absolute-path-to-your-config-file>:/docs-scraper/config.json \
+    getmeili/docs-scraper:v0.9.0 pipenv run ./docs_scraper config.json
+```
+
+::: note
+
+If you don't want to use Docker, here are [other ways to run the scraper](https://github.com/meilisearch/docs-scraper#installation-and-usage).
+
+:::
+
+`<your-meilisearch-host-url>` and `<your-meilisearch-api-key>` should be replaced by the credentials of the MeiliSearch instance you set up at [the first step](./search_bar_for_docs.md#run-a-meilisearch-instance).
+`<absolute-path-to-your-config-file>` should be the **absolute** path of your configuration file defined at [the previous step](./search_bar_for_docs.md#configuration-file).
+
+The API key you must provide should have the permissions to add documents into your MeiliSearch instance. Thus, you need to provide the private key or the master key.
```

Comment:
I think you should say the private key. I don't think the master key should be used in a script or wherever. As I see it, it should be used to create the public/private key pair, and that's all, IMO

Reply:
Agreed